_posts/2016-07-28-community-testing-day-on-20160805.md

```diff
@@ -0,0 +1,120 @@
+
+# Automate tests
+
+The Pulp QE team uses a python-based testing framework to automate testing features and and regression testing. This framework is called Pulp Smash, and if you're interested in contributing with your own automated tests for some of the new features, here's how to get started:
+
+## Install Pulp Smash
+
+Pulp Smash can be easily installed onto your system. Simply follow the official documentation found here: <http://pulp-smash.readthedocs.io/en/latest/index.html> If you want a different approach to how to install it, you can also read [this post](http://www.pulpproject.org/2016/07/05/getting-started-on-pulp-and-pulp-smash-a-pulpqe-intern-summary/) from our intern.
+
+## Run the tests
+
+Once you have Pulp Smash installed and configured, we highly recommend that you read through [this](http://pulp-smash.readthedocs.io/en/latest/about.html#contributing) for a quick introduction to the process of contributing with automated tests. Then you can:
+
+- Manually test on Pulp 2.10 beta with API/CLI features.
+- Develop Pulp Smash tests to help automated testing.
+- Choose one of the existing test cases that need to be automated here: <https://github.com/PulpQE/pulp-smash/issues>.
+
+# Reporting bugs and issues
+
+While playing with Pulp 2.10 Beta, you may encounter an issue or something that does not quite match your expectations or the existing documentation. For these cases we kindly ask that you help us by:
+
+- Opening a Redmine issue: <https://pulp.plan.io/projects/pulp/issues>
+- Opening a Pulp Smash issue: <https://github.com/PulpQE/pulp-smash/issues>
+
+# Test Day Report
+
+After the completion of Test Day, Pulp QE  will deliver a summary via email containing information about:
+
+- What new features were tested?
+- What existing features were tested?
+- Number of Redmine issues opened
+- Number of Pulp Smash issues created
+- Number of PR/automation submitted
+- Number of Redmine issues closed/reopened
+
+Also, there will be a follow-up blog post on pulpproject.org with this summary.
+
+
+# Who's available
+
+The following PulpQE members will be available on IRC/freenode to provide help and guidance with any questions you may have, as well as help with your tests, inform of known workarounds, or just chat about the awesome new features:
+
+- Chongrui Zhang (chongrui)
+- Preethi Thomas (preethi)
+
+The following  PulpQE members can help out with Pulp Smash automation as well:
+
+- Chongrui Zhang (chongrui)
+- Elyezer Rezende (elyezer)
+
+The Pulp Development members will also help out with the testing and chatting with the attendees.
+
+**We expect that Pulp 2.10 will be an amazing new release, and we look forward to seeing you joins us this Test Day!**
+
+**Pulp QE Team**
+
+
+# Reference
```

Comment:
I propose this references section can go away and these can be moved into the document itself as links.